W 11934107 rpa connector update esb (#342)
mulesoft/docs-composer: en/mulesoft/modules/ROOT/pages/ms_composer_rpa_reference.adoc

* Customer information updates +
Update customer information automatically across different systems that do not have APIs and that are accessible only through the UI.

== Invoke Short-Running RPA Processes

When invoking short-running RPA processes from Composer, invoke a single flow by choosing the *Use response in this flow* button. The RPA execution is retrieved by the RPA step and subsequent steps can use the RPA step datapills.

== Invoke Long-Running RPA Processes

When invoking RPA processes from Composer, a two-flow Composer scenario is only required when the RPA process takes longer than the allowed completion timeout period, which is three minutes.

> For long-running processes, you can configure two flows that finish within the Composer flow timeout period, which avoids blocking other events that would be queued for processing while waiting for the RPA process to complete.

Reviewer: ```suggestion
For long-running processes, you can configure two flows that finish within the Composer flow timeout period to avoid blocking other in-queue events from processing while waiting for the RPA process to complete.
```

> For the first flow, select *Poll response from another flow* and then configure a second RPA poll trigger flow that listens to executions for the process invoked by the first flow. When activating or testing the two-flow scenario, ensure that you start the second poll trigger flow first and then start the invoke flow. This way the poll trigger flow will not miss events generated by the invoke flow.

Reviewer: ```suggestion
For the first flow, select *Poll response from another flow* and then configure a second RPA poll trigger flow that listens to executions for the process invoked by the first flow. When activating or testing the two-flow scenario, ensure that you start the second poll trigger flow first and then start the invoke flow. This way, the poll trigger flow won't miss events generated by the invoke flow.
```

> If data pills from the first RPA invoke flow are needed in the second poll trigger flow, store them in an external database using the *RPA Execution ID* as the key. The data pills are then retrieved when the poll trigger polls the executions from RPA by the *RPA Execution ID*.

Reviewer: ```suggestion
If data pills from the first RPA invoke flow are needed in the second poll trigger flow, store them in an external database using the *RPA Execution ID* as the key. The data pills are then retrieved by the *RPA Execution ID* when the poll trigger polls the executions from RPA. 
```

Invokes a process in RPA. Provide a value for *RPA Process Name*. Additionally, select the activity parameters that you want to set and set values for those parameters.

> == Test RPA Process

Reviewer: Should it be Test the RPA Process or Test RPA Processes?

> You can test your RPA Process instantly without actually invoking the process.

Reviewer: ```suggestion
You can test your RPA Process instantly without invoking the process.
```
